integreat_cms/cms/forms/contacts/contact_form.py

```diff
@@ -93,6 +94,10 @@ def validate_opening_hours(self) -> list[dict[str, Any]] | None:
 
         :return: The valid opening hours
         """
+        # Remove when opening hours become available for all users
+        if not self.request.user.has_perm("cms.test_beta_features"):
```

Comment:
```suggestion
        if self.request and not self.request.user.has_perm("cms.test_beta_features"):
```
This is a little strange, but if I understand correctly the request is a 'NoneType' for tests. Therefore the test is failing at the moment. I think this should do it. At least it does for me locally :thinking:

Reply:
Thank you for speedy reply 😸 

Hmm, I'm not for adjusting the code for the test and did otherway around 🤔 Could you check the second commit?